charts/internal/machine-controller-manager/seed/templates/deployment.yaml

```diff
@@ -39,7 +39,7 @@ spec:
       serviceAccountName: machine-controller-manager
       terminationGracePeriodSeconds: 5
       containers:
-      - name: openstack-machine-controller-manager
+      - name: machine-controller-manager
```

Comment:
I am not a logging/metrics expert, but changing the container names hopefully doesn't affect metric/log collection. If you aren't sure, i would suggest to revert it to `openstack-machine-controller-manager` and we can probably relook at metric improvements in a second round.